charts/kube-prometheus-stack/charts/crds/crds/crd-alertmanagerconfigs.yaml

```diff
@@ -8,6 +7,18 @@ metadata:
     operator.prometheus.io/version: 0.73.2
   name: alertmanagerconfigs.monitoring.coreos.com
 spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          name: prometheus-operator-admission-webhook
```

Comment:
I got this from here: https://prometheus-operator.dev/docs/user-guides/webhook/#converting-alertmanagerconfig-resources

However, I guess this can not be never a valid configuration for kube-prometheus-stack. Such settings needs to be templated, but helm doesn't allow any templating here.

Any ideas to solve the issue?

Reply:
This will probably need to be patched from a hook since we can never know the name of the chart or we decide on a fixed name :/.

For the first option, if someone is using tools to manage these deployments the change will have to be ignored.

Reply:
And if people apply CRD updates after upgrade, they may miss that, too.

Reply:
What about an static service name?

Reply:
I agree that is the safer option.